cli/popper/log.py

```diff
@@ -34,12 +34,12 @@ class PopperFormatter(logging.Formatter):
     BOLD_RED = '[01;31m'
 
     log_format = {
-        'DEBUG':       '{}%(levelname)s: %(msg)s {}'.format(BOLD_CYAN, RESET),
+        'DEBUG':       f'{BOLD_CYAN}%(levelname)s: %(msg)s {RESET}',
```

Comment:
thanks @edeediong! Minor comment: the `%` can also be changed for f-strings

Reply:
Oh, I didn't know that. I'm gonna do that.

Reply:
@ivotron I saw https://docs.python.org/3/library/logging.html#logging.Formatter and https://stackoverflow.com/questions/56269302/how-to-setup-python-logging-format-using-f-string-style. But I'm still yet to understand how to format the loggers to f string

Reply:
that's ok. I'll merge as is. thanks!